docs/data/data-grid/demo/demo.md

```diff
@@ -6,7 +6,7 @@ title: Data Grid - Demo
 
 <p class="description">Use the demo below to explore the available features.</p>
 
-{{"demo": "FullFeaturedDemo.js", "defaultCodeOpen": false, "bg": "inline"}}
+{{"demo": "OneMasterDemo.js", "defaultCodeOpen": false, "bg": "inline"}}
```

Comment:
I don't think someone will ever click the "Open in CodeSandbox" button, also because it doesn't work with demos with more than one file 😁, so better to hide it. The `.tsx` can also be deleted, since only `.js` is used.

```suggestion
{{"demo": "OneMasterDemo.js", "defaultCodeOpen": false, "bg": "inline", "hideToolbar": true}}
```